src/ui/public/styles/base.less

```diff
@@ -254,6 +254,22 @@ kbn-table, .kbn-table {
     }
   }
 
+  td {
+    position: relative;
+    .table-cell-filter {
+      position: absolute;
+      background-color: rgba(255, 255, 255, 0.75);
+      white-space:nowrap;
+      right: 0;
+      visibility: hidden;
+    }
+  }
+
+  td:hover {
+    .table-cell-filter {
+      visibility: visible;
+    }
+  }
```

Comment:
Could you move these styles to `ui/public/doc_table/doc_table.less` and nest everything inside of the `discover-table-row` class? This should scope everything to just the Discover table.

```less
.discover-table-row {
  td {
    position: relative;

    .table-cell-filter {
      position: absolute;
      background-color: rgba(255, 255, 255, 0.75);
      white-space:nowrap;
      right: 0;
      visibility: hidden;
    }

    &:hover {
      .table-cell-filter {
        visibility: visible;
      }
    }
  }
}
```